linkis-dist/pom.xml

```diff
@@ -218,4 +218,87 @@
+                                            <arg value="--build-arg" /> <arg value="LINKIS_HOME=${linkis.home}" />
+                                            <arg value="--build-arg" /> <arg value="LINKIS_CONF_DIR=${linkis.conf.dir}" />
+                                            <arg value="--build-arg" /> <arg value="LINKIS_LOG_DIR=${linkis.log.dir}" />
+                                            <arg value="${project.build.directory}" />
+                                        </exec>
+                                    </target>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>build-linkis-web-image</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <target name="linkis-web-image" if="linkis.build.web">
+                                        <echo message="Building linkis web image ..." />
+                                        <mkdir dir="${basedir}/target/apache-linkis-web-${project.version}-dist" />
+                                        <copy todir="${basedir}/target/apache-linkis-web-${project.version}-dist">
+                                            <fileset dir="${basedir}/../web/dist" includes="**" />
+                                        </copy>
+                                        <exec executable="docker" failonerror="true" >
+                                            <arg value="build" />
+                                            <arg value="-f"          /> <arg value="${project.basedir}/docker/Dockerfile" />
+                                            <arg value="-t"          /> <arg value="${project.parent.artifactId}-web:${project.version}" />
+                                            <arg value="--target"    /> <arg value="linkis-web" />
+                                            <arg value="--build-arg" /> <arg value="IMAGE_BASE_WEB=${image.base.web}" />
+                                            <arg value="--build-arg" /> <arg value="LINKIS_VERSION=${project.version}" />
+                                            <arg value="--build-arg" /> <arg value="LINKIS_HOME=${linkis.home}" />
+                                            <arg value="${project.build.directory}" />
+                                        </exec>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
 </project>
```

Comment:
Emmm, this seems to be a legacy code, but would you help add an empty line in the end of the file. Thanks.